contracts/smart-account/modules/MultiOwnedECDSAModule.sol

```diff
@@ -84,11 +88,15 @@ contract MultiOwnedECDSAModule is
             revert OwnerAlreadyUsedForSmartAccount(owner, msg.sender);
 
         _smartAccountOwners[owner][msg.sender] = true;
-        numberOfOwners[msg.sender]++;
+        unchecked {
+           ++numberOfOwners[msg.sender];
+        }
     }
 
     /// @inheritdoc IMultiOwnedECDSAModule
     function removeOwner(address owner) external override {
+        if(!_smartAccountOwners[owner][msg.sender]) 
+            revert NotAnOwner(owner, msg.sender);
         _transferOwnership(msg.sender, owner, address(0));
         --numberOfOwners[msg.sender];
```

Comment:
Since we have the check above we can also uncheck the decrementation?

Reply:
fixed